nixos/modules/services/networking/seafile.nix

```diff
@@ -1,36 +1,64 @@
-{ config, lib, pkgs, ... }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 with lib;
 let
   cfg = config.services.seafile;
   settingsFormat = pkgs.formats.ini { };
 
-  ccnetConf = settingsFormat.generate "ccnet.conf" cfg.ccnetSettings;
+  ccnetConf = settingsFormat.generate "ccnet.conf" (
+    cfg.ccnetSettings
+    // {
+      Database = {
```

Comment:
We should probably swap this to allow end users to overwrite parts of the config. Also I think we should use lib.recursiveUpdate just to be safe.